CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 - Updated hap-nodejs to [0.9.7](https://github.com/homebridge/HAP-NodeJS/releases/tag/v0.9.7) (bug fixes)
 - Updated hap-nodejs to [0.9.8](https://github.com/homebridge/HAP-NodeJS/releases/tag/v0.9.8) (bug fixes)
 - Updated hap-nodejs to [0.10.0](https://github.com/homebridge/HAP-NodeJS/releases/tag/v0.10.0) (features)
+- Updated hap-nodejs to [0.10.1](https://github.com/homebridge/HAP-NodeJS/releases/tag/v0.10.0) (changes)
```

Comment:
@Shaquu, @sjorge means there is an error in the link to version 0.10.1.
Old: `(https://github.com/homebridge/HAP-NodeJS/releases/tag/v0.10.0)`
New: `(https://github.com/homebridge/HAP-NodeJS/releases/tag/v0.10.1)`